.travis.yml

```diff
@@ -39,3 +39,20 @@ before_script:
 script:
   - if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then ./venv/bin/black . --check; fi
   - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then ./venv/bin/python test-cli.py default; fi
+  - mysql -e "CREATE DATABASE codalab_bundles;"
```

Comment:
The steps that are not directly the test but that are part of the setup are moved to the `before_script` section of this file. As you can see above we already have all the commands like `CREATE DATABASE` there. Please don't re-do them here. 

As far as I can tell we just need to add another line to run your worker tests like the following:
`if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then ./test-worker.sh; fi`

The reason to use if statements is to only run tests on Python 2 (and only run the format checker on Python3)